ROMFS/px4fmu_common/init.d/airframes/5001_quad_+

```diff
@@ -1,26 +1,28 @@
 #!/bin/sh
 #
-# @name Generic 10" Quad + geometry
+# @name Generic Quad + geometry
 #
 # @type Quadrotor +
 # @class Copter
 #
-# @maintainer Lorenz Meier <lorenz@px4.io>
-#
 # @board bitcraze_crazyflie exclude
 # @board px4_fmu-v2 exclude
 #
 
 . ${R}etc/init.d/rc.mc_defaults
 
 param set-default CA_ROTOR_COUNT 4
-param set-default CA_ROTOR0_PX 0.15
-param set-default CA_ROTOR0_PY 0.15
-param set-default CA_ROTOR0_KM -0.05
-param set-default CA_ROTOR1_PX -0.15
-param set-default CA_ROTOR1_PY -0.15
-param set-default CA_ROTOR1_KM -0.05
-param set-default CA_ROTOR2_PX 0.15
-param set-default CA_ROTOR2_PY -0.15
-param set-default CA_ROTOR3_PX -0.15
-param set-default CA_ROTOR3_PY 0.15
+
```

Comment:
Note: This was incorrectly also doing a Quad X. I corrected according to https://docs.px4.io/main/en/airframes/airframe_reference.html#quadrotor